redis/commands/core.py

```diff
@@ -75,14 +75,14 @@ def to_bytes(val: Union[str, bytes, memoryview]):
 def get_from_cache(rds, key: str, get_cmd: str = 'GET') -> Tuple[Any, bool]:
     """
     Gets key from cache if exists,
-    Otherwise gets key from redis, saves in the cache and returns
+    Otherwise gets key from redis(if get_cmd is not None), saves in the cache and returns
     """
     if not rds._is_pipeline and rds._cache_write_once:
         key = to_str(key)
         key_type = get_key_type(key, rds._key_types)
         if key_type == KeyCacheProp.WRITE_ONCE:
             val = rds._key_cache.get(key, None)
-            if val is None:
+            if val is None and get_cmd is not None:
                 val = rds.execute_command(get_cmd, key)
                 rds._key_cache[key] = val
```

Comment:
If `get_cmd is None` we should still execute `rds.execute_command(get_cmd, key)`? Maybe just skip saving it in cache? Otherwise `hexists` may say that the key does not exist, when it actually does exists?

Reply:
If `get_cmd` is None and the key wasn't already there, this function returns None. 
In hexists function if we get None from the cache, the we are calling hexists in redis.